doc/MANUAL.adoc

```diff
@@ -726,6 +726,14 @@ safe to use, but not all file systems support the feature. For such file
 systems, ccache will fall back to use plain copying (or hard links if
 <<config_hard_link,*hard_link*>> is enabled).
 
+[#config_generate_compilation_database]
+*generate_compilation_database* (*CCACHE_GENERATE_COMPILATION_DATABASE* or *CCACHE_NOGENERATE_COMPILATION_DATABASE*, see _<<Boolean values>>_ above)::
+
+    If true, `ccache` writes a compilation database fragment for any compiler.
+    The output will be written next to the object file with a `.cdb.json`
```

Comment:
Would it make sense not to use a `.json` extension since the file is not a valid JSON file?